javascript/ql/lib/semmle/javascript/security/dataflow/TaintedPathCustomizations.qll

```diff
@@ -80,499 +132,596 @@ module TaintedPath {
       predicate isAbsolute() { relativeness = "absolute" }
 
       /** Gets the path label with normalized flag set to true. */
-      PosixPath toNormalized() {
+      PathLabel toNormalized() {
```

Comment:
I think all of these utility predicates should preserve the platform 
(add `this.getPlatform() = result.getPlatform()` to `toNormalized`, `toNonNormalized`, `toAbsolute`, toRelative`).

Reply:
I think your `NormalizingPathCall` doesn't preserve the platform due to you not preserving the platform in these utility predicates.  
(The rest look OK).